custom/icds_reports/utils/data_pull/Oct-2019.sql

```diff
@@ -0,0 +1,126 @@
+/*
+# Total AWCs
+# AWCs Launched
+# Districts launched
+# Avg. # of Days AWCs open
+*/
+SELECT
+state_name,
+num_awcs,
+num_launched_awcs,
+num_launched_districts,
+awc_days_open,
+CASE WHEN num_launched_awcs>0 THEN awc_days_open/num_launched_awcs ELSE awc_days_open END AS average_awc_open
+FROM agg_awc_monthly WHERE aggregation_level=1 AND month='2019-10-01'
+
+
+-- # of AWCs that submitted Infra form
+SELECT state_name, sum(num_awc_infra_last_update)
+FROM agg_awc_monthly
+WHERE aggregation_level=1 AND month='2019-10-01'
+GROUP BY state_name
+
+/*
+# of AWCs that reported available drinking water
+# of AWCs that reported unavailable drinking water
+# of AWCs that reported available functional toilet
+# of AWCs that reported unavailable functional toilet
+*/
+SELECT state_name,
+count(*) FILTER (WHERE infra_clean_water IN (1,2,3,5)) AS "Available drinking water",
```

Comment:
hey its only 1. not 2,3 ,5

Reply:
Aha! you are right, I picked the values in the form.
Though this might not be correct because 5 is also a valid value if you see in the app, not changing anything there but just flagging.

Reply:
https://github.com/dimagi/commcare-hq/pull/25893/commits/8233a57d107678ccc8e3c4cd86bf77a5d219f974